src/ensembl/src/store.ts

```diff
@@ -19,21 +19,22 @@ const googleAnalyticsMiddleWare = (store: any) => (next: any) => (
   if (action.meta && action.meta.ga && action.meta.ga.category) {
     // The action and category fields are mandatory
     GoogleAnalyticsTracking.trackEvent(action);
-  } else if (action.type.indexOf('LOCATION_CHANGE') != -1) {
+  } else if (action.type === LOCATION_CHANGE) {
     // If the location history has been changed, track it as a pageview
     GoogleAnalyticsTracking.trackPageView(
       action.payload.location.pathname +
         action.payload.location.search +
         action.payload.location.hash
     );
+    return false;
```

Comment:
Why the `return false`? Don’t we want to call `next` to let this action reach the reducer eventually?

Reply:
That's removed now. I added it while investigating the double rendering issue.